### ISSUE TITLE
Issues with reopening Hashes file

### ISSUE THREAD

**Issue by zrksyd** [ScanMountGoat/ArcExplorer#25]
I've figured out that when I copy my Hashes.txt from CrossArc and paste it into ArcExplorer's directory, it works fine. But next time I open it, it'll say that it cannot read Hashes.txt. This issue can be fixed by doing the same process by copying from CrossArc but doing this every time is very impractical.

**Comment by ScanMountGoat** [comment]
Does the hashes file included with ArcExplorer work? You should be able to update hashes in ArcExplorer itself. I'm not sure what the use case for copying the hashes manually would be.

**Comment by zrksyd** [comment]
Yes it works the first time. Clicking update in the update popup gives me another identical error.
![image](https://user-images.githubusercontent.com/6856627/112790371-34481780-902d-11eb-8f96-baee4dda357f.png)

**Comment by ScanMountGoat** [comment]
How are you launching the program? That should only be an issue if it can't find the file at the appropriate path.

**Comment by zrksyd** [comment]
Just a regular double click on the EXE. The Hashes.txt is in the same directory as the EXE.

**Comment by ScanMountGoat** [comment]
I was able to reproduce a similar issue where the application couldn't find the hashes, log, or preferences. Let me know if this fixes the issue. 
[abs_path_fix_windows_x64.zip](https://github.com/ScanMountGoat/ArcExplorer/files/6225041/abs_path_fix_windows_x64.zip)

**Comment by zrksyd** [comment]
That build does fix the issue.
